scripts/cppcheck_in_makefile.sh

```diff
@@ -41,7 +41,7 @@ function cpp_check
 {
   dir=$1
   echo "cppcheck $dir"
-  cppcheck --error-exitcode=42 -j 8 --enable=all -I src/lib -I test/ -I test/unittests $dir > /dev/null 2> /tmp/cppcheck.log
+  cppcheck -ULM_OFF --error-exitcode=42 -j 8 --enable=all -I src/lib -I test/ -I test/unittests $dir > /dev/null 2> /tmp/cppcheck.log
```

Comment:
Maybe a comment explaining why this addiotion is needed would be useful. Something like "We need -ULM_OFF in order cppcheck to process on LM_X macros etc etc"